compiler/plugins/target/AMD-AIE/aie/Passes.td

```diff
@@ -0,0 +1,22 @@
+// Copyright 2025 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#ifndef AMDAIE_AIE_PASSES
+#define AMDAIE_AIE_PASSES
+
+include "mlir/Pass/PassBase.td"
+
+def AMDAIECoreToStandard :
+    Pass<"amdaie-standard-lowering", "ModuleOp"> {
```

Comment:
Is it okay to change the name of the pass flag to be `iree-amdaie-core-to-standard` and accordingly change the lit test file name from `local_locks.mlir` -> `core-to-standard.mlir`?

I'm okay with having this addressed later and not part of this PR.